apps/mobile/package.json

```diff
@@ -19,52 +19,52 @@
     "@karakeep/shared-react": "workspace:^0.1.0",
     "@karakeep/trpc": "workspace:^0.1.0",
     "@react-native-menu/menu": "^1.1.6",
-    "@tanstack/react-query": "^5.67.1",
+    "@tanstack/react-query": "^5.69.0",
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.0",
-    "expo": "~52.0.11",
-    "expo-build-properties": "^0.13.1",
-    "expo-checkbox": "^4.0.0",
-    "expo-clipboard": "^7.0.0",
-    "expo-constants": "~17.0.3",
-    "expo-dev-client": "^5.0.4",
-    "expo-haptics": "^14.0.0",
-    "expo-image": "^2.0.2",
-    "expo-image-picker": "^16.0.3",
-    "expo-linking": "~7.0.3",
-    "expo-navigation-bar": "~4.0.4",
-    "expo-router": "~4.0.9",
-    "expo-secure-store": "^14.0.0",
+    "expo": "~52.0.46",
+    "expo-build-properties": "^0.13.3",
+    "expo-checkbox": "^4.0.1",
+    "expo-clipboard": "^7.0.1",
+    "expo-constants": "~17.0.8",
+    "expo-dev-client": "^5.0.20",
+    "expo-haptics": "^14.0.1",
+    "expo-image": "^2.0.7",
+    "expo-image-picker": "^16.0.6",
+    "expo-linking": "~7.0.5",
+    "expo-navigation-bar": "^4.0.9",
+    "expo-router": "~4.0.21",
+    "expo-secure-store": "^14.0.1",
     "expo-share-intent": "3.0.0",
-    "expo-status-bar": "~2.0.0",
-    "expo-system-ui": "^4.0.4",
-    "expo-web-browser": "^14.0.1",
+    "expo-status-bar": "~2.0.1",
+    "expo-system-ui": "^4.0.9",
+    "expo-web-browser": "^14.0.2",
     "lucide-react-native": "^0.354.0",
     "nativewind": "^4.1.23",
     "react": "^18.3.1",
-    "react-native": "0.76.3",
+    "react-native": "0.76.9",
     "react-native-awesome-slider": "^2.5.3",
     "react-native-blob-util": "^0.21.2",
-    "react-native-gesture-handler": "~2.21.2",
+    "react-native-gesture-handler": "~2.20.2",
```

Comment:
any minor version changes in `apps/mobile` were made by migration scripts provided by react-native and expo.